rs-lib/Cargo.toml

```diff
@@ -11,8 +11,8 @@ license = "MIT"
 [dependencies]
 anyhow = { workspace = true }
 base64 = { workspace = true }
-deno_ast = { version = "0.33.2", features = ["bundler", "codegen", "proposal", "react", "sourcemap", "transforms", "typescript", "visit", "transpiling"] }
-deno_graph = { version = "0.65.0", default-features = true }
+deno_ast = { version = "0.33.3", features = ["bundler", "codegen", "proposal", "react", "sourcemap", "transforms", "typescript", "visit", "transpiling"] }
```

Comment:
Do these need to be "=0.XXX"?

Reply:
To pin them to a patch? I don't believe for these crates. I bumped it here in order to get a decorator fix.